keras/layers/core.py

```diff
@@ -80,8 +80,6 @@ def __init__(self, models, mode='sum'):
         ''' Merge the output of a list of models into a single tensor.
             mode: {'sum', 'concat'}
         '''
-        if len(models) < 2:
-            raise Exception("Please specify two or more input models to merge")
```

Comment:
Sorry, I accidentally included my old autoencoder design in this PR, this was part of it. No idea how that happened, should be all fixed now.